sdk/core/azure-core-tracing-opentelemetry/samples/README.md

```diff
@@ -0,0 +1,21 @@
+---
+page_type: sample
+languages:
+  - python
+products:
+  - azure
+  - azure-core-tracing-opentelemetry
+urlFragment: azure-core-tracing-opentelemetry-samples
+---
+
+# Azure Core Tracing Opentelemetry Integration Python Samples
+
+These code samples show using the tracing opentelemetry integration with few services including servicebus, storage and eventgrid. A simple console span exporter is used in this example to export the calls. However, the code to use the azure monitor exporter has also been provided and commented in each sample.
+
```

Comment:
We should have a note on the packages needed to be installed in order to have those samples working.